pocs/focuser/birger.py

```diff
@@ -403,33 +403,43 @@ def _initialise(self):
     def _get_serial_number(self):
         response = self._send_command('sn', response_length=1)
         self._serial_number = response[0].rstrip()
-        self.logger.debug("Got serial number {} for {} on {}".format(self.uid, self.name, self.port))
+        self.logger.debug("Got serial number {} for {} on {}".format(self.uid,
+                                                                     self.name,
+                                                                     self.port))
 
     def _get_library_version(self):
         response = self._send_command('lv', response_length=1)
         self._library_version = response[0].rstrip()
-        self.logger.debug("Got library version '{}' for {} on {}".format(self.library_version, self.name, self.port))
+        self.logger.debug("Got library version '{}' for {} on {}".format(self._library_version,
+                                                                         self.name,
+                                                                         self.port))
 
     def _get_hardware_version(self):
         response = self._send_command('hv', response_length=1)
         self._hardware_version = response[0].rstrip()
-        self.logger.debug("Got hardware version {} for {} on {}".format(self.hardware_version, self.name, self.port))
+        self.logger.debug("Got hardware version {} for {} on {}".format(self._hardware_version,
+                                                                        self.name,
+                                                                        self.port))
 
     def _get_lens_info(self):
         response = self._send_command('id', response_length=1)
         self._lens_info = response[0].rstrip()
-        self.logger.debug("Got lens info '{}' for {} on {}".format(self.lens_info, self.name, self.port))
+        self.logger.debug("Got lens info '{}' for {} on {}".format(self._lens_info,
+                                                                   self.name,
+                                                                   self.port))
 
     def _initialise_aperture(self):
```

Comment:
Since this is only used by Huntsman it doesn't matter, but I'd suggest American English spelling of words in the API shared with PANOPTES.